packages/cozy-harvest-lib/src/helpers/permissions.js

```diff
@@ -0,0 +1,7 @@
+import intersection from 'lodash/intersection'
+
+export const build = (doc, verbs) => ({
+  type: doc._type,
+  values: [doc._id],
+  verbs: intersection(verbs, ['DELETE', 'GET', 'POST', 'PUT'])
+})
```

Comment:
I do not see where this is used ?

Reply:
Oh well spotted, it has been commited by mistake. Should go in our future permission model instead. I'll remove this file from the PR right now.